octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -36,13 +47,73 @@ async def start(self, bot_id: str) -> bool:
         :return: success of the evaluator's start
         """
         await super().start(bot_id)
-        self.consumer_instance = await get_chan(MATRIX_CHANNEL, self.matrix_id).new_consumer(self.matrix_callback)
+        self.consumer_instance = await get_chan(MATRIX_CHANNEL, self.matrix_id).new_consumer(
+            self.strategy_matrix_callback)
+        return True
+
+    @classmethod
+    def get_is_requiring_complete_TA_updates(cls) -> bool:
```

Comment:
I don't think it's the best way to handle this feature. It would be better to have a `check_TA_completion` that can be overwritten or completed if necessary.